DigitalLearningSolutions.Web/ViewModels/TrackingSystem/Delegates/CourseDelegates/CourseDelegatesViewModel.cs

```diff
@@ -8,22 +8,29 @@
 
     public class CourseDelegatesViewModel
     {
-        public CourseDelegatesViewModel(CourseDelegatesData courseDelegatesData)
+        public CourseDelegatesViewModel(
+            CourseDelegatesData courseDelegatesData,
+            string sortBy,
+            string sortDirection,
+            string? filterBy,
+            int page
+        )
         {
             CustomisationId = courseDelegatesData.CustomisationId;
 
             var courseOptions = courseDelegatesData.Courses
                 .Select(c => (c.CustomisationId, c.CourseNameWithInactiveFlag));
             Courses = SelectListHelper.MapOptionsToSelectListItems(courseOptions, courseDelegatesData.CustomisationId);
 
-            // TODO: HEEDLS-564 - paginate properly instead of taking 10.
-            var delegates = courseDelegatesData.Delegates.Take(10)
-                .Select(cd => new SearchableCourseDelegateViewModel(cd));
             CourseDetails = courseDelegatesData.CustomisationId.HasValue
                 ? new SelectedCourseDetails(
-                    courseDelegatesData.CustomisationId.Value,
-                    courseDelegatesData.Courses,
-                    delegates
+                    courseDelegatesData,
+                    sortBy,
+                    sortDirection,
+                    filterBy,
+                    page,
+                    new Dictionary<string, string>
+                        { { "customisationId", courseDelegatesData.CustomisationId.Value.ToString() } }

```

Comment:
Can we get the controller to specify the key as `"customisationId"`, since that's where the parameter you're trying to match is?